PROCESSORS.md

```diff
@@ -73,6 +74,7 @@ def printer(num_rows=10, last_rows=None, fields=None, resources=None,
 - `resources` - optional, allows to limit the printed resources, same semantics as `load` processor `resources` argument
 - `header_print` - optional, callable used to print each resource header
 - `table_print` - optional, callable used to print the table data
+- `max_cell_size` - optional, limit the maximum cell size to the given number of bytes, the underlying tabulate library cannot handle large data sizes
```

Comment:
bytes is a bit unintuitive... why go with that?

Reply:
we could change it to boolean - `allow_large_cells` or something like that and hide that complexity

or, convert to str and check len